scripts/004_Banding_TestData.sql

```diff
@@ -0,0 +1,35 @@
+USE Team1_RyanB;
+DELIMITER $$
+DROP PROCEDURE IF EXISTS add_bandingTestData_004 $$
+CREATE PROCEDURE add_bandingTestData_004()
+BEGIN
+
+START TRANSACTION;
+
+-- Insert test data into Capability table
+INSERT INTO Banding(`level_of_band`)
+VALUES('Manager'),
+('Executive');
+INSERT INTO Banding(`training_available`)
+VALUES('Yes'),
+('No');
```

Comment:
Take a look at the Banding table in the database. I suspect that separating these out means that we have 4 bands every time this procedure runs. 2 with levels and no training info, 2 with training info but no levels.